openstudio-standards/test/test_primary_school.rb

```diff
@@ -7,8 +7,8 @@ class TestPrimarySchool < CreateDOEPrototypeBuildingTest
 
   building_types = ['PrimarySchool']
   
-  templates = ['DOE Ref Pre-1980','DOE Ref 1980-2004','90.1-2010']
-  climate_zones = ['ASHRAE 169-2006-2A','ASHRAE 169-2006-3B','ASHRAE 169-2006-4A','ASHRAE 169-2006-5A']
+  templates = ['DOE Ref Pre-1980']
```

Comment:
Were these turned off for runtime or something?  I'd still like to test these combos.